--- a/etc/designate/designate.conf.sample
+++ b/etc/designate/designate.conf.sample
@@ -9,10 +9,10 @@
 debug = False
 
 # Top-level directory for maintaining designate's state
-#state_path = /var/lib/designate
+state_path = /var/lib/designate
 
 # Log directory
-#logdir = /var/log/designate
+logdir = /var/log/designate
 
 # Driver used for issuing notifications
 #notification_driver = designate.openstack.common.notifier.rabbit_notifier
 