--- a/etc/moniker.conf.sample
+++ b/etc/moniker.conf.sample
@@ -9,10 +9,10 @@
 debug = False
 
 # Top-level directory for maintaining moniker's state
-#state_path = /var/lib/moniker
+state_path = /var/lib/moniker
 
 # Log directory
-#logdir = /var/log/moniker
+logdir = /var/log/moniker
 
 # Driver used for issuing notifications
 #notification_driver = moniker.openstack.common.notifier.rabbit_notifier
 