--- a/etc/moniker.conf.sample
+++ b/etc/moniker.conf.sample
@@ -7,10 +7,10 @@ verbose = True
 debug = False
 
 # Top-level directory for maintaining moniker's state
-#state_path = /var/lib/moniker
+state_path = /var/lib/moniker
 
 # Log directory
-#logdir=/var/log/moniker
+logdir=/var/log/moniker
 
 # There has to be a better way to set these defaults
 allowed_rpc_exception_modules = moniker.exceptions, moniker.openstack.common.exception
 